### PR TITLE
Added 2014-12-29-the-death-of-cocoa.md's translation

### PR DESCRIPTION
**Pull request by LiLejia**

### CODE REVIEW

**Comment by LiLejia** [comment]
@croath 求斧正

File: 2014-12-29-the-death-of-cocoa.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
+---
+layout: post
+title: The Death of Cocoa
+category: ""
+author: Mattt Thompson
+translator: Henry Lee
+excerpt: "对于我们大多数而言，苹果的简洁、优雅和它软硬件的结合是我们为什么在这个平台开发的原因，尽管如此，即使Swift才开始被使用几个月，Cocoa已经开始失去它原有的光芒了。"
+---
+Cocoa实际就是Objective-C的标准库，包含了许多用语言编写程序最需要的一些框架，例如Foundation、AppKit和Core Data。而Cocoa Touch基本上是用UIKit替换AppKit，而且它与Cocoa在很多系统框架都是能互换的。
```
Comment (croath) [inline 2014-12-29-the-death-of-cocoa.md]: 中英文之间加个空格呗？

Comment (rickytan) [inline 2014-12-29-the-death-of-cocoa.md]: 顶楼上，就应该加空格

**Comment by mattt** [comment]
Thanks for your hard work on this, @LiLejia!

@croath @rickytan: Could you please let me know when this is ready to go out?

**Comment by LiLejia** [comment]
Fixed the space issue @croath @rickytan

**Comment by croath** [comment]
@mattt Yep of course we'll keep working on reviewing these sort of translation pull requests. And by the time the review is over and the pull request is merged, it will be a good time to ship it :)

**Comment by croath** [comment]
@LiLejia cocoa 之死这篇都翻译了，把 cocoa 永生那篇也翻译了吧。。。

**Comment by LiLejia** [comment]
@croath Ok

**Comment by natecook1000** [comment]
Thanks again, @LiLejia, @croath, & @rickytan — up on the site now! :tada:

**Comment by croath** [comment]
:clap:

**Comment by LiLejia** [comment]
My pleasure.

**Pull request merged**